### PR TITLE
design: installer registry configurator design doc

### PR DESCRIPTION
**Pull request by colhom**
\cc @robszumski

### CODE REVIEW

File: Documentation/design/registry-configuration.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    node_agent                      = "quay-lab.tectonic.dev/tectonic-mirror/node-agent:787844277099e8c10d617c3c807244fc9f873e46"
+    prometheus_operator             = "quay-lab.tectonic.dev/tectonic-mirror/prometheus-operator:v0.9.1"
+    node_exporter                   = "quay-lab.tectonic.dev/tectonic-mirror/node-exporter:v0.14.0"
+...
+
+```
+
+The most common case where this pattern arises is when installing Tectonic in an environment with restricted or non-existent public internet access (egress).
+
+In any case, the assumption is made that there is at least a single image registry accessible from where Tectonic is being installed. This can is generally a Quay Enterprise or Artifactory instance.
```
Comment (robszumski) [inline Documentation/design/registry-configuration.md]: a Quay Enterprise **or other registry.**

**Comment by colhom** [comment]
@robszumski sorry for delay, PTAL again

**Comment by cpanato** [comment]
@robszumski can you check if this PR is still valid? I guess this should move to https://github.com/coreos/tectonic-docs, right?

**Comment by cpanato** [comment]
closing due to inactivity. thanks for your PR. if you think this is still useful, please re submit.

**Pull request closed without merging**